build/automation/var/platform-texas/v1/account-live-k8s-prod.mk

```diff
@@ -5,6 +5,16 @@ AWS_WAF_ACL_ID = ff530a4e-689d-4d90-b3ab-ac2160b5863d
 
 TF_VAR_terraform_platform_state_store = nhsd-texasplatform-terraform-state-store-lk8s-$(AWS_ACCOUNT_NAME)
 
```

Comment:
For prod, vars are:
TF_VAR_dos_replica_db := core-dos-sf-replica.dos-db-rds
TF_VAR_dos_security_group := live-lk8s-prod-uec-sf-core-dos-sf-replica-sg
TF_VAR_dos_read_replica_secret_name := core-dos/deployment
TF_VAR_ccg_etl_source_db := pathwaysdos